extension/textext/requirements_check.py

```diff
@@ -572,7 +575,7 @@ def find_tkinter(self):
 
         return RequirementCheckResult(True, ["TkInter is found"])
 
-    def find_ghostscript(self, version=None):
+    def find_ghostscript(self, version=None, op=operator.eq):
```

Comment:
Instead of adding extra parameter I would change `version` to `version_spec` and require it to be PEP440 complaint. 

Usage example `self.find_ghostscript(">1.0,<2.0")`. 

But since the `requirement_check` is going to be removed along with port to 1.0, I don't think we should spend extra time on it

Reply:
I know what you mean. Thank you! Since this will be used only for a limited amount of time I will keep it and modify it later as an exercise :memo: for me.